### ISSUE TITLE
Standardise format of Accumulo Store unit tests

### ISSUE THREAD

**Issue by a09631** [GovernmentCommunicationsHeadquarters/Gaffer#191]
Currently there are several different formats for the tests used by the Accumulo Store.  These should be standardised before refactoring of the store begins.  Has to be done before gh-190

**Comment by p013570** [comment]
Merged into develop.
